libs/ledger-live-common/src/families/cosmos/crypto/Juno.ts

```diff
@@ -0,0 +1,54 @@
+import { makeLRUCache } from "../../../cache";
+import { CosmosRewardsState } from "../types";
+import cosmosBase from "./cosmosBase";
+
+class Juno extends cosmosBase {
+  lcd: string;
+  stakingDocUrl: string;
+  unbonding_period: number;
+  constructor() {
+    super();
+    this.lcd = "https://lcd-juno.itastakers.com";
+    this.stakingDocUrl =
+      "https://support.ledger.com/hc/en-us/articles/6235986236957-Earn-Osmosis-OSMO-staking-rewards-in-Ledger-Live?docs=true";
+    this.default_gas = 100000;
+    this.unbonding_period = 28;
+  }
+  public getRewardsState(): any {
+    return makeLRUCache(
+      async () => {
+        // Hardcoded mock values
+        const targetBondedRatio = 0.1;
+        const assumedTimePerBlock = 7;
```

Comment:
can you add the unit in comment ?
```suggestion
        const assumedTimePerBlock = 7; // min ? sec ? days ?
```

Reply:
fixed. I've changed the variable same